From ffd89bfc08a92f987e407059c00d9df49c5bc358 Mon Sep 17 00:00:00 2001
From: Huang Chaofan <chaofan.huang@nxp.com>
Date: Thu, 29 Mar 2018 19:33:47 +0800
Subject: [PATCH 3555/5242] MLK-17946[IMX8QXP B0] Change VPU_PIX_FMT_ASP to
 V4L2_PIX_FMT_MPEG4

commit  1a068699bb798dffbb6604a5cc973b74eb944356 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change VPU_PIX_FMT_ASP to V4L2_PIX_FMT_MPEG4 for vpu b0 encoder and
decoder

Signed-off-by: Huang Chaofan <chaofan.huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu-decoder-b0/vpu_b0.c         |    2 +-
 drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mxc/vpu-decoder-b0/vpu_b0.c b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
index ec5d796..5067e97 100755
--- a/drivers/mxc/vpu-decoder-b0/vpu_b0.c
+++ b/drivers/mxc/vpu-decoder-b0/vpu_b0.c
@@ -247,7 +247,7 @@ static void vpu_log_shared_mem(struct vpu_ctx *ctx)
 	},
 	{
 		.name       = "MPEG4 ASP Encoded Stream",
-		.fourcc     = VPU_PIX_FMT_ASP,
+		.fourcc     = V4L2_PIX_FMT_MPEG4,
 		.num_planes = 1,
 		.vdec_std   = VPU_VIDEO_ASP,
 	},
diff --git a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
index dd6ba17..ad9df20 100644
--- a/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
+++ b/drivers/mxc/vpu-encoder-b0/vpu_encoder_b0.c
@@ -90,7 +90,7 @@
 	},
 	{
 		.name       = "MPEG4 ASP Encoded Stream",
-		.fourcc     = VPU_PIX_FMT_ASP,
+		.fourcc     = V4L2_PIX_FMT_MPEG4,
 		.num_planes = 1,
 		.venc_std   = VPU_VIDEO_ASP,
 	},
-- 
1.7.9.5

